packages/angular/ssr/src/routes/route-config.ts

```diff
@@ -15,17 +15,14 @@ import { EnvironmentProviders, InjectionToken, makeEnvironmentProviders } from '
  * @developerPreview
  */
 export enum RenderMode {
-  /** AppShell rendering mode, typically used for pre-rendered shells of the application. */
-  AppShell,
-
   /** Server-Side Rendering (SSR) mode, where content is rendered on the server for each request. */
-  Server,
+  Server = 1,
 
   /** Client-Side Rendering (CSR) mode, where content is rendered on the client side in the browser. */
-  Client,
+  Client = 2,
 
   /** Static Site Generation (SSG) mode, where content is pre-rendered at build time and served as static files. */
-  Prerender,
+  Prerender = 3,
```

Comment:
Question: Do we need to maintain theses indexes? Should we re-index on 0?

Reply:
We could